raiden/raidenwebui/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
No, not required, just created automatically by the tool. Already removed and merged again current master for PR' merge.